cypress/integration/smokeTests/sharedTests/waf.ts

```diff
@@ -73,7 +73,7 @@ describe('Test AWS Managed Common Rule Set', () => {
   // clients shouldn't read or run. Example patterns include extensions like .log and .ini.
   it('Try a URL that would break if RestrictedExtensions_URIPATH were turned on', () => {
     const longUrl =
-      '/api/ga4gh/trs/v2/tools/%23workflow%2Fgithub.com%2Fnf-core%2Fhlatyping/versions/1.1.2/plain-NFL/descriptor//nextflow.config';
```

Comment:
This is a little odd if you needed it to pass. The type here should be case insensitive and allow both underscore and dashes

For example
```
~/dockstore-tool-registry-service-schemas$ curl -X 'GET'   'http://phoenix:4200/api/ga4gh/trs/v2/tools/registry.hub.docker.com%2Fsevenbridges%2Fpcawg_bwa_sbg_modified/versions/1.0/plain-cwl/descriptor'   -H 'accept: application/json' | md5sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8503    0  8503    0     0   414k      0 --:--:-- --:--:-- --:--:--  415k
ba0205d2450045caf65f6fd3e00d4d9c  -
~/dockstore-tool-registry-service-schemas$ curl -X 'GET'   'http://phoenix:4200/api/ga4gh/trs/v2/tools/registry.hub.docker.com%2Fsevenbridges%2Fpcawg_bwa_sbg_modified/versions/1.0/PLAIN-CWL/descriptor'   -H 'accept: application/json' | md5sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8503    0  8503    0     0   406k      0 --:--:-- --:--:-- --:--:--  415k
ba0205d2450045caf65f6fd3e00d4d9c  -
~/dockstore-tool-registry-service-schemas$ curl -X 'GET'   'http://phoenix:4200/api/ga4gh/trs/v2/tools/registry.hub.docker.com%2Fsevenbridges%2Fpcawg_bwa_sbg_modified/versions/1.0/PLAIN_CWL/descriptor'   -H 'accept: application/json' | md5sum
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8503    0  8503    0     0   491k      0 --:--:-- --:--:-- --:--:--  518k
ba0205d2450045caf65f6fd3e00d4d9c  -
```